card/mod/carrierwave/lib/carrier_wave/file_card_uploader.rb

```diff
@@ -38,14 +16,139 @@ def full_filename for_file
     end
   end
 
+  # Takes care of the file upload for cards with attached files.
+  # Most of the upload behaviour depends on the card itself.
+  # (e.g. card type and storage option chosen for the card). So in contrary
+  # to CarrierWave's default uploader we depend very much on the model
+  # (= card object) to get the correct paths for retrieving and storing
+  # the file.
+  #
+  # Cards that support attachments (by default those are cards of type "file"
+  # and "image") accept a file handle as a card attribute.
+  #
+  # @example Attaching a file to a file card
+  #   Card.create name: "file card", type: :file,
+  #               file: File.new(path_to_file)
+  #
+  # @example Attaching a image to a image card
+  #   Card.create name: "file card", type: :image,
+  #               image: File.new(path_to_image)
```

Comment:
with this clear articulation, I'm wondering for a moment whether this is the ideal API, or whether developers would find it easier to remember if it were always (`file:`)

(no need to change anything now, just a thought)